Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
 name = "hamlet"
-version = "0.2.0"
+version = "0.2.1"
 authors = [
     "Wim Looman <wim@nemo157.com>",
     "critiqjo <john.ch.fr@gmail.com>"
 ]
 description = "Html token definitions for stream processing"
 
 readme = "README.md"
+homepage = "https://nemo157.com/hamlet"
 repository = "https://github.com/Nemo157/hamlet"
+documentation = "https://nemo157.com/hamlet/docs/0.2.1/hamlet"
```

Comment:
But this link will have to be updated every time, right? I adding the homepage is enough. May be update the README.md to include a link to the homepage as well...